test/unit/plugins/translation-inPlugins_test.js

```diff
@@ -1,262 +0,0 @@
-                },
-                {
-                    name: 'dimming',
-                    type: 'Percentage',
-                    value: '87'
-                }
-            ];
-
-            function testMiddleware(entity, typeInformation, callback) {
-                entity.contextElements[0].attributes[1].value = entity.contextElements[0].attributes[1].value + '%';
-                callback(null, entity, typeInformation);
-            }
-
-            iotAgentLib.addUpdateMiddleware(testMiddleware);
-
-            iotAgentLib.update('light1', 'Light', '', values, function (error) {
-                should.not.exist(error);
-                contextBrokerMock.done();
-                done();
-            });
-        });
-    });
-
-    describe('When a new query translation middleware is added to the IoT Agent', function () {
-        const attributes = ['state', 'dimming'];
-
-        beforeEach(function () {
-            nock.cleanAll();
-
-            contextBrokerMock = nock('http://192.168.1.1:1026')
-                .matchHeader('fiware-service', 'smartgondor')
-                .matchHeader('fiware-servicepath', 'gardens')
-                .post(
-                    '/v1/queryContext',
-                    utils.readExampleFile('./test/unit/examples/contextRequests/queryContext1.json')
-                )
-                .reply(200, utils.readExampleFile('./test/unit/examples/contextResponses/queryContext1Success.json'));
-        });
-
-        it('should call the middleware', function (done) {
-            let called = false;
-
-            function testMiddleware(entity, typeInformation, callback) {
-                entity.contextResponses[0].contextElement.attributes[1].value =
-                    entity.contextResponses[0].contextElement.attributes[1].value + '%';
-
-                called = true;
-
-                callback(null, entity, typeInformation);
-            }
-
-            iotAgentLib.addQueryMiddleware(testMiddleware);
-
-            iotAgentLib.query('light1', 'Light', '', attributes, function (error, result) {
-                should.not.exist(error);
-                called.should.equal(true);
-                done();
-            });
-        });
-        it('should call the middleware', function (done) {
-            function testMiddleware(entity, typeInformation, callback) {
-                entity.contextResponses[0].contextElement.attributes[1].value =
-                    entity.contextResponses[0].contextElement.attributes[1].value + '%';
-
-                callback(null, entity, typeInformation);
-            }
-
-            iotAgentLib.addQueryMiddleware(testMiddleware);
-
-            iotAgentLib.query('light1', 'Light', '', attributes, function (error, result) {
-                should.not.exist(error);
-                result.contextResponses[0].contextElement.attributes[1].value.should.equal('23%');
-                done();
-            });
-        });
-    });
-});
```

Comment:
This file does not have an equivalent in NGSI-v2 folder, so it is needed to translate it to v2 version.